src/sentry/static/sentry/app/views/eventsV2/querycard.tsx

```diff
@@ -52,6 +53,10 @@ const StyledQueryCard = styled(Link)`
   &:active {
     box-shadow: 0px 0px 0px 6px rgba(209, 202, 216, 0.5);
   }
+
+  * {
```

Comment:
@doralchan putting this in so that the graph appears clickable with respect to the entire card.

Reply:
@dashed Can you add this as a comment within the code?

Reply:
Added